### PR TITLE
Fix the error when running Yolox evaluation on COCO test-dev

### PR DESCRIPTION
**Pull request by shenyi0220**
This allows the following command to be run without error when database and config files are deployed:
"python -m yolox.tools.eval -n  yolox-s -c yolox_s.pth -b 64 --conf 0.001 -d 1 --test"

### CODE REVIEW

File: yolox/exp/yolox_base.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def __init__(self):
         # name of annotation file for evaluation
         self.val_ann = "instances_val2017.json"
         # name of annotation file for testing
-        self.test_ann = "instances_test2017.json"
+        self.test_ann = "image_info_test-dev2017.json"
```
Comment (FateScript) [inline yolox/exp/yolox_base.py:46]: `instances_test2017.json` is used for coco-test. Why modifiying it here?  You could use cli to modify it for your need.

Comment (shenyi0220) [reply yolox/exp/yolox_base.py]: Sure, I updated the patch and pls check it again

**Comment by FateScript** [comment]
Thanks for your contribution. LGTM.

**Comment by shenyi0220** [comment]
> Thanks for your contribution. LGTM.

@FateScript So has it been good enough to merge?

**Comment by FateScript** [comment]
> > Thanks for your contribution. LGTM.
> 
> @FateScript So has it been good enough to merge?

Don't worry, I will merge it.

**Pull request merged**